pkg/proxysql/client_test.go

```diff
@@ -51,8 +51,8 @@ func TestProxySQLGetDB(t *testing.T) {
 		if err == nil {
 			t.Fatal("expected error for failed connection")
 		}
-		if !strings.HasSuffix(err.Error(), "no such host") {
-			t.Fatalf("expected a 'no such host' error, got %v", err)
+		if !strings.HasSuffix(err.Error(), "no such host") && !strings.HasSuffix(err.Error(), "i/o timeout") {
```

Comment:
Is this change meant to be in this PR?

Reply:
This is to fix a test failure when run in Actions, as the invalid host lookup now times out instead of returning "no such host" (but the important thing for this test is that connecting to an invalid hostname fails, which is still the case with this change). I've not dug further into why this has changed.